### ISSUE TITLE
Create Tic-Tac-Toe game from Scratch or just improve the existing code.

### ISSUE THREAD

**Issue by Axe-Pearl** [Axe-Pearl/Tic-Tac-Toe#1]
Follow these steps to make your pull request accepted - 

1. Star the Repository
2. Complete the Task
3. Make a Pull request
4. I will merge your Pull request

**Comment by Aspireve** [comment]
Hey!! @Axe-Pearl Could I work on this? This seems like something I can do

**Comment by Axe-Pearl** [comment]
@Aspireve Yes you can for sure.

**Comment by Aspireve** [comment]
I guess what I could add to a game is levels, an easy (with random moves), a medium with a little more better predictions and a hard mode with more predictions.
I could use the minimax algorithm to apply it
Here is a video to help explain it: [Algorithms Explained – minimax and alpha-beta pruning](https://www.youtube.com/watch?v=l-hh51ncgDI)
Should I go ahead with this?

**Comment by Axe-Pearl** [comment]
@Aspireve Yes, It will be great if you make the game more interesting by putting the levels and using Algorithms.

**Comment by Aspireve** [comment]
Yup sure!!
But I'm also kinda a beginner so it'll take me a bit more time

**Comment by aanchalsharmaa1024** [comment]
I know both the algorithms (minimax and Alpha-beta pruning)
Could u plz assign it to me?

**Comment by aanchalsharmaa1024** [comment]
I am also interesting in making the Tic-Tac-Toe game from scratch .....plz assign the issue to me

**Comment by Soumyajit2825** [comment]
@Axe-Pearl i also want to contribute in this project

**Comment by Aspireve** [comment]
Yup just give me some more time and I'll make the PR (Just understood its implementation

**Comment by Aspireve** [comment]
Hey @Axe-Pearl, I tried to solve it but am not able to understand why it isn't working, since its taking me so long i guess you could assign it to some-one else, sorry for wasting so much of your time

**Comment by Axe-Pearl** [comment]
@aanchalsharmaa1024 Sure, you can also work on the same. I can assign it to multiple people.
Check this issue also for explaining the Algorithm - https://github.com/Axe-Pearl/Tic-Tac-Toe/issues/3

**Comment by Axe-Pearl** [comment]
@Soumyajit2825 Sure, you can also work on the same. I can assign it to multiple people.
Check this issue also for explaining the Algorithm - https://github.com/Axe-Pearl/Tic-Tac-Toe/issues/3

**Comment by Axe-Pearl** [comment]
Hey @Aspireve , yes take your time. Can you mention which technology you working?

**Comment by Axe-Pearl** [comment]
@Aspireve It's okay, You can check others issues as well I have made.

**Comment by Soumyajit2825** [comment]
@Axe-Pearl i have done the PR on issue #4

**Comment by Soumyajit2825** [comment]
@Axe-Pearl can you please add this repository under Hacktoberfest 2k22?

**Comment by Axe-Pearl** [comment]
@Soumyajit2825 Done but if you contributing for Hacktoberfest then the tags needed are **Hacktoberfest** or 
**Hacktoberfest- Accepted**, check this snippet from Hacktoberfest Website-> https://hacktoberfest.com/participation/

![image](https://user-images.githubusercontent.com/66417880/194043877-8a469d00-dd8c-4186-b582-3fc5ccb594f5.png)

**Comment by praptisharma28** [comment]
I have already made a beautiful TIC -TAC-TOE app with audio included, it's even one of the recent repository on my profile. Can you please assign me this so that I can generate a PR for you to merge.

**Comment by Axe-Pearl** [comment]
@praptisharma28 , you can start working on it

**Comment by Pratik7897** [comment]
Hey @Axe-Pearl! I'd love to work on this. 
I'll build the Tic-Tac-Toe game using HTML, CSS & JS with Minimax Algorithm.
